pubspec.yaml

```
@@ -29,6 +29,7 @@ dependencies:
   go_router: ^14.2.8
   local_auth: ^2.3.0
   path_provider: ^2.1.5
+  crypto: ^3.0.6
 
```

pin dependency mismatched version of intl to temporarily fix test pipeline
```suggestion

dependency_overrides:
  intl: ^0.20.2
  
```